src/org/parosproxy/paros/extension/report/ReportGenerator.java

```diff
@@ -121,12 +124,13 @@ public static File stringToHtml(String inxml, String infilexsl, String outfilena
 				logger.error(e.getMessage(), e);
 				// Save the xml for diagnosing the problem
 				BufferedWriter bw = null;
-	
+				showDialogForGUI();
 				try {
 					bw = new BufferedWriter(new FileWriter(outfilename + "-orig.xml"));
 					bw.write(inxml);
 				} catch (IOException e2) {
-					logger.error(e.getMessage(), e);
+					logger.error("Failed to write debug XML file", e);
+					return outfile;

```

Comment:
- It should return "new File(outfilename)" instead, to keep the old behaviour.
- In the outer catch block it should return the same.